Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We can mount your directory ".m2" with VOLUME "/root/.m2"
+#  -v "~/.m2:/root/.m2"
+#
+# We must mount your project directory with "/root/incubator-zeppelin"
+#  -v "$(path project zeppelin):/root/incubator-zeppelin"
+#
+FROM centos:latest
+
+MAINTAINER Zeppelin Project (incubating) 
+
+#for plugin frontend-maven-plugin we need maven >= 3.1.0
+ENV MAVEN_VERSION=3.3.3 
+ENV MAVEN_HOME=/usr/local/maven 
+ENV MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=256m"
+
+ENV PATH=${PATH}:${MAVEN_HOME}/bin
+
+VOLUME "/root/.m2"
+VOLUME "/root/incubator-zeppelin"
+
+EXPOSE 8080
+
+#bzip2 for npm (tar .bz2)
+RUN yum -y update && yum install -y git bzip2 && yum clean all
+
+RUN curl http://download.oracle.com/otn-pub/java/jdk/7u80-b15/jdk-7u80-linux-x64.rpm -L -C - -b "oraclelicense=accept-securebackup-cookie" -o /tmp/jdk.rpm && \
+    rpm -ivh /tmp/jdk.rpm && rm -f /tmp/jdk.rpm
+
+RUN curl -L http://wwwftp.ciril.fr/pub/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz -o /tmp/apache-maven.tar.gz && \
```

Comment:
I create line per tools/command for unless layer's docker but i can split by commands.